### DISCUSSION TITLE
Add a List filter to Stars

### DISCUSSION THREAD

**Discussion by neptoo** [community/community#20423] [Other Feature Feedback, Questions, & Ideas]
I'd want to filter my stars by "My list" or "Not in list" beside the "Type" and "Language" buttons.
It will be easier to find and organize stars without needing to go into the List.

**Comment by radar-blip** [comment]
Thats a splendid idea! 👍🏼

**Comment by Amer1u** [comment]
I need help if you don't mind pls I preper my new app it's about whatsapp hacking can you support me if you now pls

**Comment by jfullstackdev** [comment]
you can create your own list at the Stars tab 

![image](https://user-images.githubusercontent.com/47092464/180000013-bfa0dc83-501f-4d79-89bd-a5e5ac30d794.png)

**Comment by VladimirSeleznev** [comment]
Vladimir.nikolaevich.seleznev@gmail.com
